Sources/Java2SwiftLib/JavaTranslator.swift

```diff
@@ -383,6 +391,20 @@ extension JavaTranslator {
 
     topLevelDecls.append(classDecl)
 
+    let subClassDecls = javaClass.getClasses().compactMap {
+      $0.flatMap { clazz in
+        do {
+          return try translateClass(clazz)
+        } catch {
+          logUntranslated("Unable to translate '\(fullName)' subclass '\(clazz.getName())': \(error)")
+          return nil
+        }
+      }
+    }.flatMap(\.self)
+
+    topLevelDecls.append(
+      contentsOf: subClassDecls
+    )
```

Comment:
Hmm. I don't think we actually want the recursion here? As noted above, I think nested classes that are translated should be listed in the top-level Java2Swift.config. Then, the top-level loop that calls `translateClass` will also handle nested classes.